### DISCUSSION TITLE
can someone help me answer this ?

### DISCUSSION THREAD

**Discussion by juleedev** [coreybutler/nvm-windows#711] [Support] [answered]
Problem

I need to install nvm for windows 10 and have it work for my admin profile and my non admin profile. When I do all my programming it's with my regular user account. But to install things / admin stuff, I have to provide an admin account.

I downloaded and ran the latest installer from this repo.  
When I try to run the installer, I have to use my admin account. Windows prompts me for it and after I supply the creds, the setup completes without any issues. The problem is that nvm only works when I run a command window as my admin user ... but I need to be able to run nvm as my regular non priv. user as well.

So right now, when I open a "command" window as admin, i can run

```
Microsoft Windows [Version 10.0.19043.1348]
(c) Microsoft Corporation. All rights reserved.

C:\WINDOWS\system32>nvm --version

Running version 1.1.9.
```
But if I run command just as my regular user,

```
Microsoft Windows [Version 10.0.19043.1348]
(c) Microsoft Corporation. All rights reserved.

C:\Users\me>nvm --version
'nvm' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\me>
```

In the environment variables - for the admin user - the path has been updated  But also I have them in the system environment variables. 
Any tips would be appreciated.

**Comment by coreybutler** [comment, marked as answer]
It sounds like NVM4W was installed just for the admin account. Your non-privileged account doesn't appear to be recognizing the system environment variables.

The easiest fix is to create `NVM_SYMLINK` and `NVM_HOME` _user_ environment variables. These would be the same values as they are for the admin account. [This section](https://github.com/coreybutler/nvm-windows/wiki#manual-installation) of the wiki provides details.

**Reply by juleedev** [reply]
I used the registry editor and added the two variables for the regular user profile.  Can't use control panel cuz it always makes me use the admin account to get into it in the first place.  And so when I do edit the user variables, i can only edit the admin account.  But thanks for the tip
